### ISSUE TITLE
Assume that campsites with backcountry=yes have no facililties

### ISSUE THREAD

**Issue by rhhsm** [streetcomplete/StreetComplete#5714]
We camped on a couple of backcountry campsites, and I was surprised that SC asked if there were showers, power supply and toilets.

I think SC should assume that `tourism=camp_site` with `backcountry=yes` are "without facilities" (quoting https://wiki.openstreetmap.org/wiki/Key:backcountry) and not ask for them. I didn't check if backcountry campsites are included in the Camp Type quest: I think they shouldn't be because it can be safely assumed that for such campsites, only tents are allowed (maybe caravans and camper vans are not disallowed, but "The access is usually restricted to foot, bicycle, canoe or ski" so they can't get there anyway).

I still think it is a good idea to ask for availability of drinking water though, as there may be natural sources of drinking water available for such sites.

**Comment by mnalis** [comment]
(See also recently opened: https://github.com/streetcomplete/StreetComplete/discussions/5712)

> I think SC should assume that `tourism=camp_site` with `backcountry=yes` are "without facilities"

Isn't a main purpose of SC somewhat contrary to "assume stuff", but rather to use the opportunity to actually **verify and fill-in explicit information** found on the ground, even the information that might be implied but can now be confirmed with 100% accuracy (e.g. _"well of course almost all of the major streets in city are `lit=yes`, right?"_), and even to re-ask information that was previously 100% correct but _might_ be stale or incorrect now (i.e. all resurvey quests)? We just try to **avoid being too spammy** as that likely would be a turn-off to majority of the users (well, if the user didn't mind the needless repetition, OSM would benefit by asking to **resurvey all details  every 14 days** - like that other popular mobile editor does).


I think this was final conclusion (and reasoning) about my implementation back then: https://github.com/streetcomplete/StreetComplete/pull/4213#issuecomment-1211343632, specifically:

> Ah yes. I've had a mind about disabling quests for them, but after more careful reading of https://wiki.openstreetmap.org/wiki/Key:backcountry I've decided to still ask for them, as it seems at least some of them are still possible (just less likely, i.e. assumed no by default). And as `backcountry=yes` is only a small subset of camps (according to [taginfo](https://taginfo.openstreetmap.org/tags/tourism=camp_site#combinations)) and camps are relatively rare themselves, I do not think continuing to ask for them would be too spammy. But if there is consensus to disable some of them for `backcountry=yes`, I'm not against.

(nobody tried to discuss it back then, so it went in in that state)

> I think SC should assume that `tourism=camp_site` with `backcountry=yes` are "without facilities" (quoting https://wiki.openstreetmap.org/wiki/Key:backcountry) and not ask for them


Just to be clear, wiki for [backcountry=yes](https://wiki.openstreetmap.org/wiki/Key:backcountry) also follows that _"without facilities"_ immediately with _"Occasionally such sites can have access to drinking water, compost toilets, waste disposal etc."_  


So **I guess what backcountry camp provides may vary wildly depending on the country**? :shrug: 

I wonder what do you think would be **benefits** of **not** asking the quests, though? Do you think it bothers the users too much?  Wastes too much of their time? Rare novel quests aren't fun at all? That there is significant chance they'll provide wrong information?  Something else?

I'd guess that majority of SC users wouldn't even know what a `backcountry=yes` tag is, whether it was tagged on some OSM element prior to them being presented the quest (or afterwards), and even much less how such different would affect quest mechanics. In fact, I'd hazard a guess that **most of SC users would probably consider it logical to be asked all those camp-related quests all all camp sites**.

Looking at [taginfo for backcountry=yes](https://taginfo.openstreetmap.org/tags/backcountry=yes#combinations) it seems to be used together with **9.67%** `toilets=yes`, **9.39%** `drinking_water`, **7.65%** `shower` - so to me it still seems reasonable to ask them (even if I've personally never met some of those configurations - I've only tent-camped in barely over half dozen different countries, most of them closeby Croatia - other mappers seem to have found and tagged them. That is the beauty of OSM, every now and then I find some amazing fact how different countries work in totally unimaginable (to me) ways. Or when I get to surprise them, even for obvious things (to me) like that [picnic tables benches might have backrests](https://wiki.openstreetmap.org/wiki/File:Picnic_table_with_backrest_in_Croatia.jpg))

One of the camping quests that **didn't** make it to  taginfo minimum (1000 uses?) seems to be `power_supply=yes`; that one indeed seems to be rather rare (but not non-existent) on `backcountry=yes`, which makes sense to me. Perhaps that one might be disabled? 

But I see no big harm if left in either - people will answer it has no electricity in vast majority of the cases so it will just become explicit; but camping sites (especially only ones filtered on `backcountry=yes`) are not so common as say street crossings that the quest would be considered spammy IMHO. 
And in cases when one does answer `yes`, it is even more valuable information. Also, note specifically that it might've been incorrectly tagged as `backcountry=yes`[^1], and collecting on-the-site information about other facilities will provide useful on-the-ground verified information, which might be used in addition (or to override) other tags.

---
<details>
<summary>also, it would help  if people actually used GitHub templates...</summary>


e.g. both _"feature requests"_ (like I guess this is?) and _"bug reports"_ (as perhaps you might consider it?) contain several very useful hints, and tries the streamline a way of thinking and reporting in an easier to understand format.

> **Use case**
> Provide a clear and concise description of *your use case* and what you thus think is missing, and why.
>
> **Proposed Solution**
> There may be several different solutions for your idea or use case, add your deliberations about it here to help sort out the best one(s).
> Add screenshots or mockups to visualize your idea, if applicable.

Perhaps it would've allowed me to better understand the **actual issue** (and not [XY problem](https://en.wikipedia.org/wiki/XY_problem)), would allow me to write shorter reply without some of the requests for clarifications :sweat_smile: 


</details>

[^1]: for example, [this](https://www.openstreetmap.org/node/9074026605) according to linked website might be considered `backcountry=yes` by some (it is unmanned, only gets visited by cleanup crew twice a week in the season) and yet it contains among other things _"2 lighting poles with power contacts"_ (if automated translate is not lying to me :smiley:). But even if one disagrees that it is `backcountry=yes` , that seems like even more reason to ask the camping quests, right? Perhaps best would be to suggest [OSMose](https://wiki.openstreetmap.org/wiki/Osmose) or similar check to mark camping sites with both `backcountry=yes` and `power_supply=yes` as possibly suspicious?

**Comment by matkoniecz** [comment]
Note that camp sites are relatively rare in non-dense feature areas so I would be finer with less dense quests.

And at least toilets, at least in Poland would be quite likely present even for the most isolated camp sites.

**Comment by westnordost** [comment]
@mnalis Could you put the tldr at the beginning and make it shorter in the future? (Or just write exclusively the tldr)

Thanks @matkoniecz , makes sense, closing as wnf.

**Comment by mnalis** [comment]
**TL;DR:** I'll put `TL;DR` at the beginning in the future and _try_ to make it shorter.  Would using `<details>` tag help at all?

---
It did end up too big, sorry. 

`<details>` would only help  if one renders markdown (github web interace does, but E-mail probably not)?

As an explanation; I just feel compelled not to just say equivalent of _"I don't think it's a good idea"_, unless I **explain** my views and provide **references** why I think those view are valid. And if there are several suggestions/claims, I feel need to address them (instead if ignoring them)... For three main reasons:

- I don't feel entitled that people should take my claims as granted without me providing supporting data. (In fact, it would make me feel uncomfortable and stressed if people did that, as I'd feel obliged to triple-check everything and make sure I didn't miss any data source or POV etc)

- It should help a lot in cases when I do overlook some facts or get some of them wrong -- people can then correct me and perhaps completely different conclusion can be reached. And those cases are not nearly so rare as I'd like...

- Especially as _rejection_ resulting from such answer is probably going to hurt the person suggesting the idea, I'd at least like to offer them evidence I really tried to make it work out  :heart:

**Comment by rhhsm** [comment]
Thanks @mnalis for your extensive replies, I really appreciate this! Indeed it "hurts" when issues are closed while there are still undiscussed (lengthy) posts that authors have put effort in writing...

My main reason to suggest not to ask these quests on these campsites is to avoid unnecessary addition of `*=no` tags. There  have been complaints that SC is adding too many of these, so I think we should at least avoid adding them where `*=no` is (close to) the default. The lit quest is not among those because street lighting varies widely across the world. Cycle ways in Italy is a border case (#5527). Power supply, showers and sanitary dump stations on backcountry campsites are clearly within the "quests where the answer can be assumed to always be the same (i.e. “no”) with a relative certainty" range, according to me (I learnt that toilets are much more common on them than I thought). If they are present, it is quite likely that the campsite is wrongly tagged with `backcountry=yes`, like the Estonian campsite you mentioned (it has place for 50 tents, showers, electricity, a car park, near a village, ...).

SC users probably don't encounter many backcountry campsites during survey so this issue is of minor importance, though. But it's the principle...

**Comment by mnalis** [comment]
**TL;DR:** implied values like `no` have their use, `backcountry` could've been better defined, and (last but not the least) maintainer work is much appreciated and we should try our best to reduce their workload :heart: 

---

> I really appreciate this! Indeed it "hurts" when issues are closed while there are still undiscussed (lengthy) posts that authors have put effort in writing...

Thanks, it means a lot to me it is appreciated. 

I must note however that I absolutely **see the maintainers point of view** too - they _have_ to read all the issues on their project, and still have time to do actual programming of new features, hunting bugs, etc.  It takes enormous amount of time and energy, and we users never show enough appreciation for all that volunteer work. If they also have to do other work for living, having popular project is overwhelming, to say the least. 
So my thanks goes to them  (no explicit mentions in order not to generate even more reading work for them) :heart: 

I will try to put `TL;DR` at the beginning of my longer posts in order to help them skip over, and we should all try to take effort to reduce load on them, so the project might live long and prosper! :vulcan_salute: 

Also, especially for possibly controversial subjects, perhaps we should use GitHub Discussions (or other channels like Community forum) instead of Issues, and open _**Issue**_ (linking to that discussion) only when there is **community consensus about clearly defined _action_ which should be taken**. 
Of course, it is not always easy to know when the issue might be controversial... :man_shrugging: 

- See also https://github.com/streetcomplete/StreetComplete/discussions/3450 for information how we may help reduce maintainers load.

---

> My main reason to suggest not to ask these quests on these campsites is to avoid unnecessary addition of *=no tags

I understand. However I still think that [FAQ: Why does StreetComplete often tag the absence of features?](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Why_does_StreetComplete_often_tag_the_absence_of_features%3F) and especially more detailed https://github.com/streetcomplete/StreetComplete/issues/5527#issuecomment-1987000896 explain well why tagging those implied values  is more preferable to alternatives. Especially in cases like `backcountry` which is (as you note) quite rare, so the _"bloat"_ argument doesn't really float.

---

That is especially the case for [backcountry=yes](https://wiki.openstreetmap.org/wiki/Key:backcountry) which seems not very well defined (it seems to have several different ambiguous uses) - which is unfortunately common problem with tags invented via [ATYL](https://wiki.openstreetmap.org/wiki/Any_tags_you_like) instead of following [Proposal process](https://wiki.openstreetmap.org/wiki/Proposal_process) which usually catches such issues _before_ the tag is approved, so tags produces in that way are usually clearly and unambiguously defined from the start (by adding more detailed and clear-cut explanation, and providing examples what qualifies and what does not -- and why). It is more work up-front, though, so it is understandable why people prefer to skip that, but it pays off hundredfold if tag becomes popular. There is just no way single pair of eyes can catch all possible issues the tag might have.

But, as they say, there is no point in crying over spilled milk - we cannot retroactively fix _"backcountry"_; that cat is out of the bag now... What we **can** do is **add** _**other** (more clearly defined) **tags**_ to **explicitly** quantify what exactly is or is not present. Thus my [conclusion](https://github.com/streetcomplete/StreetComplete/pull/4213#issuecomment-1211343632) that those quests should be asked even on `backcountry=yes` tagged campsites.

And if the tagging of `backcountry` turns out to be suspicious (e.g. https://overpass-turbo.eu/s/1O1E or similar), than we can use [changeset discussions](https://wiki.openstreetmap.org/wiki/Changeset#Changeset_Discussions) to reach out to mapper and ask they what _they_ meant by their use of `backcountry=yes`, and then:
 - eventually document those usages on wiki if pattern emerges
 - suggest [Osmose](https://wiki.openstreetmap.org/wiki/Osmose) QA check for easier detection of suspicious backcountry campsites (which [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) supports, BTW!)
 - [propose](https://wiki.openstreetmap.org/wiki/Proposal_process)  better tag (or tags) and maybe even eventually [deprecate](https://wiki.openstreetmap.org/wiki/Deprecation_process) `backcountry` if it turns out to be too ambiguous
